fingertip/plugins/os/centos-stream.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+import fingertip.machine
+from fingertip.util import path
+
+#[Repos]
+#http://ftp.fi.muni.cz/pub/linux/centos/8-stream/
+#official mirror: http://mirror.centos.org/centos/8-stream/
+
+REPO = 'http://mirror.centos.org/centos/'
+
+def main(m=None, fips=False):
+    m = m or fingertip.build('backend.qemu')
+    if hasattr(m, 'qemu'):
+        return m.apply(install_in_qemu, fips=fips)
+    raise NotImplementedError()
+
+
+def install_in_qemu(m=None,
+                    fips=False):
+    """
+    :param bool fips: if True, install system in FIPS mode
+    """
+
+    m = m or fingertip.machine.build('backend.qemu')
+    URL = 'http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/'
```

Comment:
Please move the constant to the top of the file outside of the functions (and maybe the comment will become unnecessary).